plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -355,18 +508,57 @@ let state_type ir =
   in
   pstr_type Nonrecursive [ td ]
 
-let cmd_type config ir =
-  let constructors = List.map (cmd_constructor config) ir.values in
+let cmd_type ir =
+  let constructors = List.map cmd_constructor ir.values in
   let td =
     type_declaration ~name:(noloc "cmd") ~params:[] ~cstrs:[]
       ~kind:(Ptype_variant constructors) ~private_:Public ~manifest:None
   in
   pstr_type Nonrecursive [ { td with ptype_attributes = [ show_attribute ] } ]
 
+let init_state config ir =
```

Comment:
I didn’t understand what this function was doing without looking at the test results, a little comment describing what will be generated would help.